contracts/rbn-staking/IncentivizedVotingLockup.sol

```diff
@@ -356,11 +407,27 @@ contract IncentivisedVotingLockup is
     LockAction _action
   ) internal {
     LockedBalance memory newLocked =
-      LockedBalance({amount: _oldLocked.amount, end: _oldLocked.end});
+      LockedBalance({
+        amount: _oldLocked.amount,
+        shares: _oldLocked.shares,
+        end: _oldLocked.end
+      });
+
```

Comment:
using [xsushi](https://etherscan.io/address/0x8798249c2E607446EfB7Ad49eC89dD1865Ff4272#code) as model for shares here